src/test/java/nl/tudelft/contextproject/test/TestUtil.java

```diff
@@ -143,12 +143,11 @@ public static Main setupMainForTesting() {
 	 */
 	public static void cleanupMain() {
 		Main main = Main.getInstance();
-		
 		if (isMock(main)) {
 			reset(main);
 		}
 		
-		Main.setInstance(null);
+		Main.setInstance(globalMain);
```

Comment:
Not sure if we need this change, but if tests are not failing, I'm fine with it, as it will speed up all the tests.